frontend/src/components/Cell.tsx

```diff
@@ -33,23 +53,7 @@ export const Cell = ({
           date,
           "yyyy-MM-dd"
         )}`}
-        onChange={(event: any) => {
-          //makes sure that users can only input positive numbers up to 999.99999999...
-          //with an unlimited number of decimals behind the delimiter
-          event.target.value = event.target.value
-            .replace(/[^0-9.]/g, "")
-            .replace(/^(\d{3})\d+/, "$1")
-            .replace(/(\.\d{2}).*$/, "$1")
-            .replace(/(\..*?)\..*/, "$1");
-          onCellUpdate({
-            id: entryId,
-            issue_id: topic.issue.id,
-            activity_id: topic.activity.id,
-            hours: event.target.value,
```

Comment:
Why did you change this from `hours` to `event.target.value`? Don't really understand it.